### DISCUSSION TITLE
Columns only show one value for multi-value fields

### DISCUSSION THREAD

**Discussion by matthewstick** [craftcms/cms#8708] [Ideas]
### Description

For fields that can have multi-values (like categories or tags) the views in the CMS just show the first entry.  This makes it seem like a bug.  

This causes my clients to get confused, thinking they just selected one item.  They then edit the item and check  to see if something is missed.  You can't scan a view and see all the values checked.  

These should at least show up as a perhaps a CSV list of values.  Or if the issue is showing all of the values, you can at least show the first X amount or show and put some ....s to truncate it and indicate that there are more.  If this is a setting, it could at least be a setting at the field level to indicate how many entries you want to show in the view.

In any case, the current model feels like  mistake and as a result I need to hide columns from the client for multi-value fields.

Curiously, multi-value checkboxes do show up right.  But definitely Categories and Tags do not.

### Steps to reproduce

1. Create an entry with a field that supports multiple values (like categories or tags)
2. Add that column to the view in the CMS
3. Just one value appears

### Additional info

This is an issue both in Craft 2.x and Craft 3.x

**Comment by brandonkelly** [comment]
This is expected behavior currently, but I agree it’s a bit awkward. Will mark as an enhancement request.
